psi4/src/export_mints.cc

```diff
@@ -1535,12 +1535,19 @@ void export_mints(py::module& m) {
         .def(py::init<>())
         .def("setName", &ExternalPotential::setName, "Sets the name")
         .def("addCharge", &ExternalPotential::addCharge, "Add a charge Z at (x,y,z)", "Z"_a, "x"_a, "y"_a, "z"_a)
+        .def("getCharges", &ExternalPotential::getCharges, "Get the vector of charge tuples")
```

Comment:
I'm not seeing `getCharges` in the ExternalPotential object. Perhaps this can do the work of `get_charges_xyz`, too?

Reply:
Fixed. I removed  ```get_charges_xyz```.